mlxtend/classifier/stacking_cv_classification.py

```diff
@@ -0,0 +1,270 @@
+#
+# An ensemble-learning meta-classifier for stacking
+# Authors: Reiichiro Nakano <github.com/reiinakano>
+#          Sebastian Raschka <sebastianraschka.com>
+#
+# License: BSD 3 clause
+
+from sklearn.base import BaseEstimator
+from sklearn.base import ClassifierMixin
+from sklearn.base import TransformerMixin
+from sklearn.base import clone
+from sklearn.utils.validation import check_is_fitted
+from sklearn.pipeline import _name_estimators
+from sklearn.externals import six
+import numpy as np
+from sklearn.cross_validation import StratifiedKFold
+from sklearn.cross_validation import KFold
+
+
+class StackingCVClassifier(BaseEstimator, ClassifierMixin, TransformerMixin):
+
+    """A Stacking CV classifier for scikit-learn estimators for classification.
+
+    Parameters
+    ----------
+    classifiers : array-like, shape = [n_regressors]
+        A list of classifiers.
+        Invoking the `fit` method on the `StackingClassifer` will fit clones
+        of these original classifiers that will
+        be stored in the class attribute
+        `self.clfs_`.
+    meta_classifier : object
+        The meta-classifier to be fitted on the ensemble of
+        classifiers
+    use_probas : bool (default: False)
+        If True, trains meta-classifier based on predicted probabilities
+        instead of class labels.
+    n_folds : int (default=2)
+        The number of folds used for while creating training data for the
+        meta-classifier during fitting.
+    use_features_in_secondary : bool (default: False)
+        If True, the meta-classifier will be trained both on the predictions
+        of the original classifiers and the original dataset.
+        If False, the meta-classifier will be trained only on the predictions
+        of the original classifiers.
+    stratify : bool (default: True)
+        If True, the cross-validation technique used for fitting the classifier
+        will be Stratified K-Fold.
+        If False, the cross-validation technique used will be Regular K-Fold.
+        It is highly recommended to use Stratified K-Fold.
+    verbose : int, optional (default=0)
+        Controls the verbosity of the building process.
+        - `verbose=0` (default): Prints nothing
+        - `verbose=1`: Prints the number & name of the regressor being fitted
+                       and which fold is currently being used for fitting
+        - `verbose=2`: Prints info about the parameters of the
+                       regressor being fitted
+        - `verbose>2`: Changes `verbose` param of the underlying regressor to
+           self.verbose - 2
+
+    Attributes
+    ----------
+    clfs_ : list, shape=[n_classifiers]
+        Fitted classifiers (clones of the original classifiers)
+    meta_clf_ : estimator
+        Fitted meta-classifier (clone of the original meta-estimator)
+
+    """
+    def __init__(self, classifiers, meta_classifier,
+                 use_probas=True, n_folds=2,
```

Comment:
`use_probas` should be set to `use_probas=False` in this case; otherwise, it would be inconsistent with the docstring. I think `use_probas=False` may be better than `use_probas=True` since some scikit estimators may not support predict_proba yet.